usr/src/uts/common/brand/lx/syscall/lx_prctl.c

```diff
@@ -65,8 +65,10 @@
 #define	LX_PR_SET_THP_DISABLE		41
 #define	LX_PR_GET_THP_DISABLE		42
 
+#define SECCOMP_MODE_FILTER	2
+
 long
-lx_prctl(int opt, uintptr_t data)
+lx_prctl(int opt, uintptr_t arg2, uintptr_t arg3, uintptr_t arg4, uintptr_t arg5)
```

Comment:
Oh, one more thing.  arg4 and arg5 are ignored, so either use __unused or eliminate arg4 and arg5. You can use gcc10 as a shadow compiler (see help on `./configure` for using the -s argument) to help out there.

Reply:
I removed them.